docs/build/tutorials/scaffold-eth.md

```diff
@@ -46,7 +46,8 @@ cd project-name
 // e.g  cd kaia-scaffold-example
 ```
 
-![Scaffold-ETH setup](/img/build/tutorials/sc-bootstrap.png)
+![Scaffold-ETH setup](/img/build/s1.png)
```

Comment:
The s1 image doesn't seem to look like the initial bootstrap Scaffold-ETH setup. Can you look into that? See this [section](https://docs.kaia.io/build/tutorials/scaffold-eth/#setting-up-development-environment-)